helpers/GetSessionIdFromCookie.ts

```diff
@@ -0,0 +1,9 @@
+﻿export default function getSessionIdFromCookie(cookie: string): string {
+    const sessionId = cookie.split(";")
+        .find(element => element
+            .startsWith("sessionID="))
+    if (!sessionId) {
+        return "";
+    }
+    return sessionId.split("=")[1];
+}
```

Comment:
this function is crying out for a simple little unit test!
Could you add a quick test that just passes in a hard coded cookie and checks that it can grab the right bit of it! :)